### DISCUSSION TITLE
[Bug]: Rak4631 crashing after 3-4 days

### DISCUSSION THREAD

**Discussion by robertfoss** [meshtastic/firmware#4075] [Troubleshooting]
### Category

Other

### Hardware

Rak4631

### Firmware Version

rak4631-2.3.4.ea61808

### Description

After a few days of uptime this device stops responding over bluetooth and is no longer reporting full data on meshmap.

I don't have access to any serial logs due to the long time to reproduce and the device living outside on a roof.

![image](https://github.com/meshtastic/firmware/assets/523260/605604b9-6b76-4559-820f-343e6359f4a8)

**Comment by madeofstown** [comment]
That's a pretty old build. Maybe try a newer firmware. I've had a 4631 running on the 2.3.10  firmware for about 5 days now.

**Comment by thebentern** [comment]
Yes earlier 2.3.X builds had contentious file system usage, especially on large meshes. I recommend updating.

**Comment by JimTheCowboy** [comment]
This seems to be a duplicate of #5839
Should be mostly solved with Firmware 2.5.20 beta and upwards and requires a NRF factory reset.
https://github.com/meshtastic/firmware/releases/tag/v2.5.20.4c97351

@fifieldt or should I tag somebody else? =)
